extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -256,6 +258,10 @@ private Properties getSchedulerConfigurationProperties(QuartzSupport quartzSuppo
                 props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".isClustered", "true");
                 props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".clusterCheckinInterval", "20000"); // 20 seconds
             }
+
+            if (storeClass.equals(JobStoreCMT.class.getName())) {
+                props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".nonManagedTXDataSource", dataSource);
```

Comment:
What is this used for? It wasn't used before, or?

Reply:
Yes, it was not need before since the non managed datasource was not configured. Now we need it so configure the `JobStoreCMT` class.

Reply:
I see. It was `JobStoreTX` before...